peps/pep-0719.rst

```diff
@@ -44,10 +44,10 @@ Actual:
 
 - 3.13 development begins: Monday, 2023-05-22
 - 3.13.0 alpha 1: Friday, 2023-10-13
+- 3.13.0 alpha 2: Tuesday, 2023-11-21
```

Comment:
Alternatively...

https://www.python.org/downloads/release/python-3130a2/ says 21st, but the tag https://github.com/python/cpython/releases/tag/v3.13.0a2 was the 22nd as was https://discuss.python.org/t/python-3-13-0-alpha-2/39379.

```suggestion
- 3.13.0 alpha 2: Wednesday, 2023-11-22
```